### ISSUE TITLE
docs/cmd/flux_completion_bash/ - recommended addendum

### ISSUE THREAD

**Issue by davehamer** [fluxcd/flux2#2061]
Hi Team,

I was going to open an issue on the main flux project, but think I've figured out the actual reason behind my troubles.

I'm using WSL2 with Ubuntu 20.04 LTS. I installed flux using the homebrew method (aka linuxbrew)
As a new user to Linuxbrew, I didn't realise quite how it worked in comparison to a normal package manager.

When following the instructions on https://fluxcd.io/docs/cmd/flux_completion_bash/ I found that just entering: ```command -v flux >/dev/null && . <(flux completion bash)
``` into my `.bashrc` didn't work. It'd say `command flux not found` when opening a new shell.

I found out that this was due to me installing via homebrew/linuxbrew - by default this doesn't add a PATH line until it seems after .bashrc has executed. This was fixed by adding the following line above the flux completion:

`export PATH=/home/linuxbrew/.linuxbrew/bin/:$PATH`

I know this isn't directly a project issue, and is more a linuxbrew issue, but as one of the recommended installation methods is via this, it'd be good to have this in the documentation perhaps?

Feel free to close this issue if I'm talking rubbish - just wanted to help future people understand why it didn't work out of the box.

**Comment by hiddeco** [comment]
If an addendum is added, I would like to see it documented as generic as possible. For example, about refreshing ones terminal session after install in case of unexpected errors.

This to prevent having to maintain specific instructions for an endless list of (future) package managers.

**Comment by stefanprodan** [comment]
I don't think this belongs in our docs, please report this bug to the Homebrew team.

**Comment by kingdonb** [comment]
The generic update can be "First, make sure the `flux` binary is in your path."

The Linuxbrew/Homebrew team already tells users to add this in their PATH, I'm sure I've seen it output as part of the instructions following or around installation of the `brew` command itself, or when certain casks are tapped, or whatever it's called.

The issue here is that you might not have added it yet, it could be added in your profile, which could be loaded before or after your bashrc, depending entirely on user configuration.

It is hard to add a note here that makes this easier for any user without assuming too much. I would propose adding the line above to the doc if anything, else we can close this issue without action. @davehamer do you want to submit the PR?

**Comment by devikanthhgoud** [comment]
Hi @hiddeco @kingdonb — I'd like to work on this if it's still open. I noticed the original docs/cmd/flux_completion_bash/ path no longer exists since the CLI docs were restructured; the reference page is now generated live from the Example field in cmd/flux/completion_bash.go. Would updating that Example string (adding a line like "First, make sure the flux binary is in your path.") be the correct way to address this now? Happy to open a PR once confirmed.

**Comment by kingdonb** [comment]
Rereading the initial report, I see this is a complicated change only because of what the user might have already done or not done - and because not every user will have installed using Homebrew. (And because homebrew does not require you to read the docs first.)

Did you trip over this and do you think that change would have helped you?
